### PR TITLE
adding support for standalone openstack deployment and integration with ceph

### PR DESCRIPTION
**Pull request by hmaheswa**
Signed-off-by: Hemanth Maheswarla <hmaheswa@hmaheswa.remote.csb>

pass logs:
osp 16.2 : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-iuuxf/
osp 17.0 : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-v9hbt/

### CODE REVIEW

File: tests/misc_env/install_standalone_osp.py [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+    # setting hostname with domain
+    set_hostname_with_domain(osp_node, "novalocal")
+
+    # Deploying the all-in-one Red Hat OpenStack Platform environment
+    status = deploy_standalone_osp(
+        osp_node, osp_version, username, password, set_env_vars
+    )
+
+    # permit root login which might have been disabled after openstack installation
+    set_permit_root_login(osp_node, "yes")
```
Comment (AmarnatReddy) [inline tests/misc_env/install_standalone_osp.py:129]: can we do this step as part of inventory file. if this is mandatory for all the nodes

File: suites/pacific/rgw/tier-2_rgw_test-openstack-integration.yaml [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+  - test:
+      abort-on-fail: true
+      clusters:
+        ceph:
+          config:
+            cephadm: true
+            commands:
+              - "ceph config set client.rgw.{daemon_id:rgw.1} rgw_keystone_accepted_roles admin"
+              - "ceph config set client.rgw.{daemon_id:rgw.1} rgw_keystone_admin_domain default"
+              - "ceph config set client.rgw.{daemon_id:rgw.1} rgw_keystone_admin_password Welcome123#"
```
Comment (viduship) [inline suites/pacific/rgw/tier-2_rgw_test-openstack-integration.yaml:126]: hi @hmaheswa , use rgw_keystone_admin_token instead of password.

**Comment by mkasturi18** [comment]
This PR had some improvements suggested by reviewer's. Would schedule a call to bring this PR to logical conclusion and update accordingly.

**Comment by hmaheswa** [comment]
Closing this PR as discussed but this branch code will be used for creating osp environment to verify bz's until new PR is raised and Merged

**Pull request closed without merging**